src/sentry/integrations/bitbucket_server/webhook.py

```diff
@@ -31,15 +31,7 @@ def update_repo_data(self, repo, event):
         """
 
         name_from_event = event["repository"]["project"]["key"] + "/" + event["repository"]["slug"]
-        # build the URL manually since it doesn't come back from the API in
-        # the form that we need
-        url_from_event = event["repository"]["links"]["self"][0]["href"]
```

Comment:
@mgaeta Are you sure that `links` is never in the payload?

Reply:
I guess we don't even update the URL so it doesn't matter

Reply:
The [documentation link](https://confluence.atlassian.com/bitbucketserver/event-payload-938025882.html) didn't get carried over from Jira. I don't know which version we're using but `links` is absent in this one.